scss/_variables.scss

```diff
@@ -366,7 +366,7 @@ $screen-md-max:              ($screen-lg-min - 1) !default;
 //** Number of columns in the grid.
 $grid-columns:              12 !default;
 //** Padding between columns. Gets divided in half for the left and right.
-$grid-gutter-width:         24px !default;
+$grid-gutter-width:         36px !default;
```

Comment:
[not important]

it seems like Bootstrap 3.x uses a default of `30px` (or at least what seems to be suggested) for $grid-gutter-width (ref: http://getbootstrap.com/customize)

I suppose we made a design decision for `36px` then